### PR TITLE
quassel: Add version 0.14.0

### PR DESCRIPTION
**Pull request by SonaliBendre**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
- Order of fields not followed
- Outdated SPDX License identifier
- Is this a 32bit or 64bit app?
- Do all the binaries support commandline arguments?

File: bucket/quassel.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "description": "Modern, cross-platform, distributed IRC client based on the Qt framework.",
+    "homepage": "https://quassel-irc.org",
+    "license": "GPL-2.0|GPL-3.0",
```
Comment (issaclin32) [inline bucket/quassel.json:4]:
```suggestion
    "license": "GPL-2.0-or-later|GPL-3.0-or-later",
```
Check **"Identifier"** at https://spdx.org/licenses/

https://github.com/quassel/quassel/blob/master/gpl-3.0.txt has `or any later versions` in it, therefore it is `GPL-3.0-or-later`.

File: bucket/quassel.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "version": "0.14.0",
+    "url": "https://quassel-irc.org/pub/quassel-windows-0.14.0.7z",
```
Comment (issaclin32) [inline bucket/quassel.json:6]: You can check the app is 32 or 64-bit using pelook (`scoop install pelook`)
If the app is 32-bit, then keep as it is (both 32-bit and 64-bit machine can run it)
If the app is 64-bit. then use
```
"architecture" {
    "64bit":  {
        "url": ...
        "hash": ...
    }
```

**Comment by issaclin32** [comment]
`bin` is only needed if  (1) The app is supposed to run under command-line, or (2) The app supports command-line arguments.

You can either check [readme](https://github.com/quassel/quassel/blob/master/README.md) or read the source code on main program to find it out. In the case of *quassel*, there are some command-line arguments.
```
Once you've set up Quassel, you may:

Add more users: quasselcore --add-user
Change the password of an existing user: quasselcore --change-userpass=username
See all available options: quasselcore --help
```

**Comment by issaclin32** [comment]
I updated the manifest because the author has not responded for a while.

**NOTES**:
* The app is built in 64-bit.
* **quassel** supports command-line args. See [ReadMe](https://github.com/quassel/quassel/blob/master/README.md) for example.
* *persist* is not needed because config is at `$Env:AppData\quassel-irc.org`. I tried tweaking some settings and creating profiles/connections, but `$dir\config` does not appear.

**Review by rashil2000 — CHANGES_REQUESTED** [review]
We should wrap the `$dir\\config` part in double quotes to account for spaces in path

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**